cmd/exec/operations.go

```diff
@@ -439,6 +440,82 @@ func executeWorkflow(url string) (executeResponse, error) {
 
 }
 
+func executeEvent(url string) (string, error) {
+
+	// Read input data from flag file
+	inputDataOsFile, err := os.Open(localAbsPath)
+
+	if err != nil {
+		return "", err
+	}
+
+	defer inputDataOsFile.Close()
+	byteResult, _ := ioutil.ReadAll(inputDataOsFile)
+	var event map[string]interface{}
+	json.Unmarshal([]byte(byteResult), &event)
+
+	// Read input data as bytes.buffer
+	inputData, err := safeLoadFile(localAbsPath)
+
+	if err != nil {
+		log.Fatalf("Failed to load input file: %v", err)
```

Comment:
It is usually better to return the error where possible. We do use `log.Fatalf` in some places in the CLI, but that's usually only  in the actual command handler function. Helper functions generally return errors. This makes them more reusable. 

Replace this line with `return "", fmt.Errorf("failed to load input file: %w", err)`